graphql/mutation_resolver.go

```diff
@@ -40,6 +40,13 @@ import (
 
 // BbCreateTicket is the resolver for the bbCreateTicket field.
 func (r *mutationResolver) BbCreateTicket(ctx context.Context, taskID string, execution *int) (bool, error) {
+	canModify, err := annotationPermissionHelper(ctx, taskID, execution)
+	if err != nil {
+		return false, err
+	}
+	if !canModify {
+		return false, nil
+	}
```

Comment:
I think these should send a `Forbidden` error message (but I think it's possible to remove these conditions if you use the suggestion from previous comment)